src/tasks/Microsoft.NET.Sdk.WebAssembly.Pack.Tasks/GenerateWasmBootJson.cs

```diff
@@ -176,6 +176,15 @@ public void WriteBootJson(Stream output, string entryAssemblyName)
         {
             var endpointByAsset = Endpoints.ToDictionary(e => e.GetMetadata("AssetFile"));
 
+            var lazyLoadAssembliesWithoutExtension = LazyLoadedAssemblies.ToDictionary(l =>
+            {
+                var extension = Path.GetExtension(l.ItemSpec);
+                if (extension == ".dll" || extension == Utils.WebcilInWasmExtension)
+                    return Path.GetFileNameWithoutExtension(l.ItemSpec);
```

Comment:
Can `.pdb` enter this method? Isn't it easier to flip the condition to:
```
if (string.IsNullOrEmpty(extension))
                    return l.ItemSpec;
return Path.GetFileNameWithoutExtension(l.ItemSpec);
```
so that we don't have to predict extensions?

Reply:
No PDB allow here.
We can't test for empty extension, because
- if user passes `MyApp.Lazy.dll`, the extension is `.dll`
- but if use passes `MyApp.Lazy`, then the extension is `.Lazy`